packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/VirtualMachines/components/PlanVMsDeleteButton.tsx

```diff
@@ -43,7 +46,7 @@ export const PlanVMsDeleteButton: FC<{
   return (
     <ToolbarItem>
       <VMsActionButton onClick={onClick} disabledReason={reason}>
-        {t('Remove virtual machines')}
+        {t('Delete VMs from plan')}
```

Comment:
why changed from virtual machines to VMs and added from plan?

Reply:
Remove fear of deletion and make it clear it's just affecting the plan.
Shortcut to VMs so that the button is not too long.
Btw we run text changes by UXD copy editors, but I'm open to feedback/changes